From 69c23e3ae1e1e9e9f1ee3a06a706d755a4b1de48 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?M=C3=93ZES=20=C3=81d=C3=A1m=20Istv=C3=A1n?=
 <mozes.adam.istvan@pm.me>
Date: Sat, 16 Nov 2019 09:58:28 +0100
Subject: [PATCH] fix: do not compare literal with "is not"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Python3.8 issued the following warning on each ponysay invocation:

/bin/ponysay/backend.py:294: SyntaxWarning: "is not" with a literal. Did you mean "!="?

This has been corrected here.

Signed-off-by: MÓZES Ádám István <mozes.adam.istvan@pm.me>
---
 src/backend.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/backend.py b/src/backend.py
index 2cc539a5..82ff1496 100644
--- a/src/backend.py
+++ b/src/backend.py
@@ -291,7 +291,7 @@ class Backend():
                         props = dollar[7:]
                         if len(props) > 0:
                             if ',' in props:
-                                if props[0] is not ',':
+                                if props[0] != ',':
                                     w = props[:props.index(',')]
                                 h = int(props[props.index(',') + 1:])
                             else:
-- 
2.25.1

